package.json

```diff
@@ -86,6 +85,7 @@
     "**/isomorphic-fetch/node-fetch": "^2.6.7",
     "**/remark-parse/trim": "1.0.1",
     "**/typescript": "4.7.4",
+    "acorn": "8.x",
```

Comment:
Without this yarn resolution, an EUI sub-dependency which emits code with optional chaining will throw errors in Webpack v4.

- See https://github.com/storybookjs/storybook/issues/14292#issuecomment-803811848
- https://github.com/webpack/webpack/issues/10227

(Note that the optional chaining babel plugin works for Kibana and Cypress but not for Storybook, hence the pin)